acp/upload_module.php

```diff
@@ -59,20 +59,20 @@ function main($id, $mode)
 		$phpbb_link_template = '#^(https://)www.phpbb.com/customise/db/download/([0-9]*?)(/composer|/manual)?/?(\?sid\=[a-zA-Z0-9]*?)?$#i';
 
 		// Work with objects class instead of $this.
-		objects::$cache = &$cache;
```

Comment:
@rxu could you please comment the necessity for this deletion?
Tested on phpBB 3.3.0 w/o this change, no errors.

Reply:
Honestly I don't remember already, Probably I had some issues with PHP 7.4 or something but now I failed to reproduce :)

Reply:
OK, then I'll discard this part for now.